pkg/capabilities/triggers/streams/streams_trigger-schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/smartcontractkit/chainlink-common/pkg/capabilities/triggers/streams/streams-trigger@1.0.0",
+  "description": "Streams Trigger",
+  "$defs": {
+    "feedId": {
+      "type": "string",
+      "pattern": "^0x[0-9a-f]{64}$",
+      "description": "The ID of the data feed."
+    },
+    "Feed": {
+      "type": "object",
+      "properties": {
+        "feedId": {
+          "$ref": "#/$defs/feedId"
+        },
+        "fullReport": {
+          "type": "string",
+          "contentEncoding": "base64",
+          "description": "Full report represented as bytes encoded as base64 string."
+        },
+        "reportContext": {
+          "type": "string",
+          "contentEncoding": "base64",
+          "description": "Report context represented as bytes encoded as base64 string. This is required to validate the signatures."
+        },
+        "signatures": {
+          "type": "array",
+          "description": "Signature over full report and report context represented as bytes encoded as base64 string.",
+          "items": {
+            "type": "string",
+            "contentEncoding": "base64"
+          },
+          "minItems": 1
+        },
+        "benchmarkPrice": {
+          "type": "string",
+          "contentEncoding": "base64",
+          "description": "This value is extracted from the fullReport. Benchmark price represented as bytes encoded as base64 string."
+        },
+        "observationTimestamp": {
+          "type": "integer",
+          "maximum": 9223372036854775807,
+          "minimum": 0,
+          "description": "This value is extracted from the fullReport. A unix timestamp represented as an int64 value. Timestamp is captured at the time of report creation."
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "feedId",
+        "fullReport",
+        "reportContext",
+        "signatures",
+        "benchmarkPrice",
+        "observationTimestamp"
+      ]
+    }
+  },
+  "properties": {
+    "config": {
+      "properties": {
+        "feedIds": {
+          "type": "array",
+          "description": "The IDs of the data feeds that will have their reports included in the trigger event.",
+          "items": {
+            "$ref": "#/$defs/feedId"
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "maxFrequencyMs": {
+          "type": "integer",
+          "description": "The interval in seconds after which a new trigger event is generated.",
+          "minimum": 1
+        }
+      },
+      "additionalProperties": false,
+      "type": "object",
+      "required": ["feedIds", "maxFrequencyMs"]
+    },
+    "outputs": {
+      "items": {
+        "$ref": "#/$defs/Feed"
+      },
```

Comment:
Looking for someone from Keystone to confirm, but I think this is actually an array of feeds that streams emits not a feed.  Then consensus changes to an array of array of feed.

When @DeividasK and I wrote this it felt much more natural this way, so I don't want to change it without some confirmation.